### PR TITLE
fix migrating m2w64-ucrt

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by mbargull** [comment]
Probably makes sense; but I'm not deeply in the know about the migration inner workings...
Is this essentially because we use the `*_stub`s in interim renders and as such have to direct mapping of the names?
(And the `r-base`/`r_base` I are just added out of necessity due to the override, I assume.)

**Comment by isuruf** [comment]
>  Is this essentially because we use the *_stubs in interim renders and as such have to direct mapping of the names?

Yes.

>  (And the r-base/r_base I are just added out of necessity due to the override, I assume.)

Yes

**Comment by mbargull** [comment]
kk, ty!

File: recipe/migrations/r-base44_and_m2w64-ucrt.yaml [hunk 1]
```diff
@@ -9,6 +9,12 @@ __migrator:
   longterm: True
   include_noarch: True
   pr_limit: 4
+  override_cbc_keys:
+    - r-base
+    - r_base
```
Comment (jakirkham) [inline recipe/migrations/r-base44_and_m2w64-ucrt.yaml:14]: Do we mean to have both `-` and `_`? Thought these evaluate to the same thing

Comment (mbargull) [reply recipe/migrations/r-base44_and_m2w64-ucrt.yaml:14]: It's equivalent to https://github.com/regro/cf-scripts/blob/ee87be96ed34c44a576a0a4b456dd00bd381695c/conda_forge_tick/make_migrators.py#L439 which is only done for the non-`override_cbc_keys` case.

**Pull request merged**